docs/test_data_in_production.md

```diff
@@ -19,57 +19,22 @@ Work with RDOS to ensure we're identifying representative samples. We want to co
- [note name][octave]v[velocity].wav -- e.g., “D#5v13.wav”. Release resonance notes: harm[note name][octave]v[velocity].wav -- e.g., “harmC2v2.wav”. Hammer samples: rel[
-1-88].wav (one per key) -- e.g., “rel23.wav”. Pedal samples: pedal[D/U][velocity].wav -- e.g., “pedalU2.wav” =\\u003e pedal release (U = up), velocity = 2 (quicker rele
-ase than velocity = 1).\\r\\n  This dataset is too large to download directly from this item page. You can access and download the data via Globus (See https://www.yout
-ube.com/watch?v=uf2c7Y1fiFs for instructions on how to use Globus).\",\"collection_tags\":[],\"creators\":[{\"value\":\"Trueman, Daniel\",\"name_type\":\"Personal\",\"g
-iven_name\":\"Daniel\",\"family_name\":\"Trueman\",\"identifier\":null,\"affiliations\":[],\"sequence\":1},{\"value\":\"Wang, Matthew\",\"name_type\":\"Personal\",\"giv
-en_name\":\"Matthew\",\"family_name\":\"Wang\",\"identifier\":null,\"affiliations\":[],\"sequence\":2},{\"value\":\"Villalta, Andrés\",\"name_type\":\"Personal\",\"give
-n_name\":\"Andrés\",\"family_name\":\"Villalta\",\"identifier\":null,\"affiliations\":[],\"sequence\":3},{\"value\":\"Chou, Katie\",\"name_type\":\"Personal\",\"given_n
-ame\":\"Katie\",\"family_name\":\"Chou\",\"identifier\":null,\"affiliations\":[],\"sequence\":4},{\"value\":\"Ayres, Christien\",\"name_type\":\"Personal\",\"given_name\":\"Christien\",\"family_name\":\"Ayres\",\"identifier\":null,\"affiliations\":[],\"sequence\":5}],\"resource_type\":\"Dataset\",\"resource_type_general\":\"DATASET\",\"publisher\":\"Princeton University\",\"publication_year\":\"2021\",\"ark\":\"88435/dsp015999n653h\",\"doi\":\"10.34770/r75s-9j74\",\"rights\":{\"identifier\":\"CC BY\",\"uri\":\"https://creativecommons.org/licenses/by/4.0/\",\"name\":\"Creative Commons Attribution 4.0 International\"},\"version_number\":\"1\",\"keywords\":[]}"))
-=>
-#<PDCMetadata::Resource:0x000055ad4be0e428
-...
-irb(main):003:0> work = Work.new(resource: bitklavier_resource)
-=>
-#<Work:0x000055ad4cea22a8
-...
-irb(main):004:0> work.collection = Collection.research_data
-=>
-#<Collection:0x000055ad4d2497d0
-...
-irb(main):013:0> work.created_by_user_id = User.find_by_uid('bs3097').id
-=> 3
-work.state = 'draft'
-=> "draft"
-irb(main):005:0> work.save
-=> true
-```
-
-The work should now be visible in the application.
\ No newline at end of file
+  1. Regenerate the json representations of the migration data by running:
+     ```
+     DATA_MIGRATION=true bundle exec rspec spec/system/data_migration
+     ```
+
+  2. Copy the newly created .json files to the server where you want to load them: 
+    
+    `scp tmp/data_migration/*.json deploy@p
+dc_describe_staging1.princeton.edu:/tmp`
+
+1. Then, ssh to the server where you want to create this sample work as the `deploy` user. 
+2. Run the import rake task, specifying the location of the .json files and the netid of the user they should import as:
+  ```
+  bundle exec rake works:import_works\[/path/to/json/files,bs3097]
```

Comment:
```suggestion
  bundle exec rake works:import_works\[/tmp,bs3097]
```

In the previous step, I think you identify a destination, so I think you could use that same destination here... Except there may be other files you don't want in `/tmp` ... maybe a date-stamped subdirectory just for this transfer? Or maybe the script will skip files that aren't json?

Reply:
The JSON files are not going to /tmp. They are going to a tmp/data_migration directory within the application. When they are copied to the server, they will go wherever the user decides to put them.